vertical-pod-autoscaler/e2e/v1/updater.go

```diff
@@ -278,6 +289,19 @@ func setupHamsterJob(f *framework.Framework, cpu, memory string, replicas int32)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 }
 
+func setupHamsterCronJob(f *framework.Framework, cpu, memory string, replicas int32) {
```

Comment:
Can we have this reused in recommender e2e as well?

Reply:
Resolved, function deleted from updater.go and moved to common.go